tests/public_api_tests/test_backwards_compatible.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2013 Shotgun Software Inc.
```

Comment:
I wasn't sure either. Since this was the first test I just went this route thinking we could rename the file to something proper in the future. I agree that with the name it has at the moment it sounds a one test file and one test suite.